src/utils.ts

```diff
@@ -94,6 +94,13 @@ export function idForProject(projectId: BigInt, cv: CV): string {
   return `${cv[0]}-${projectId.toString()}`;
 }
 
+export function idForVeNftContract(
```

Comment:
nice!

Reply:
@jmilldotdev since the addresses for these contracts will be unique, can we just do...?

```
export function idForVeNftContract(address: Address): string {
  return `${address.toHexString()}`;
}
```

Reply:
yup go for it